ipmikvm

```diff
@@ -49,7 +49,9 @@ if ! command -v unpack200 >/dev/null; then
         exit 1
     else
         unpack200() {
-            "$JAVA_HOME/bin/unpack200" "$@"
+            local exe="$JAVA_HOME/jre/bin/unpack200"
+            [[ ! -e "$exe" ]] && exe="$JAVA_HOME/bin/unpack200"
```

Comment:
The `[[` is a bashism: use `[` or `test`. But the rest of the script uses `test`, so use that.